### DISCUSSION TITLE
High CPU usage when trying to view video from gallery

### DISCUSSION THREAD

**Discussion by shadow82** [photoprism/photoprism#4340] [Q&A]
Hi!
When I try to watch a video from my gallery using PhotoPrism Android app I'm getting high CPU usage on the server with multiple processes opening the video file.
Android app waits stuck and server CPU skyrockets.

Anyone had such thing?

![image](https://github.com/photoprism/photoprism/assets/16290219/0a8048b0-e722-4e6f-9fce-15fc7928a364)

**Comment by DJKeterson** [comment]
Interesting find, so I went and tested this and it appears to be the case but can't determine the cause as it happens rarely, like once in 20 different videos I tried playing.
